src/Utils/Hooks/useOnboardingModal.ts

```diff
@@ -19,14 +20,17 @@ export const useOnboardingModal = () => {
   // Check to see if we should open onboarding (logged in + ?onboarding=true),
   // show it, and then immediately remove the query param
   useEffect(() => {
-    if (!isLoggedIn || !match.location.query.onboarding) return
+    if (!isLoggedIn || !match.location.query.onboarding || initialized.current)
+      return
 
     showOnboarding()
 
     router.replace({
       ...match.location,
       query: omit(match.location.query, "onboarding"),
     })
+
+    initialized.current = true
```

Comment:
This effect ran twice. Which again: shouldn't be a problem. In the case of the exclusion, it counts the referrer as `/undefined` on the second go round. Which is... interesting.